spec/models/miq_dialog_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe MiqDialog do
+  context '.seed' do
+    it 'seeds from the core with correct metadata' do
+      root = Rails.root.join('product', 'dialogs', 'miq_dialogs')
+      allow(described_class).to receive(:find_by)
```

Comment:
right, when I was trying that, rspec complained about the other calls - now it works 🤷‍♂️ 
thanks for letting me re-visit that